docs/_sources/about_reco.rst.txt

```diff
@@ -107,3 +107,18 @@ Improving the highest-ranked recommendations has a more important effect than im
 
 .. math::
     NDCG@k = \frac{1}{\left | A \right |} \sum_{i=1}^{\left | A \right |} \frac {\sum_{r=1}^{\left | P_i \right |} \frac{rel(P_{i,r})}{log_2(r+1)}}{\sum_{r=1}^{\left | A_i \right |} \frac{1}{log_2(r+1)}}
+
+Diversity Recommender Metrics
+-----------------------------
+Diversity recommender metrics evaluate the quality of recommendations in terms of the diversity of recommended items.
+
+Inter-List Diversity
+^^^^^^^^^^^^^^^^^^^^
+Inter-List Diversity@k measures the inter-list diversity of the recommendations when only k recommendations are
+made to the user. It measures how user's lists of recommendations are different from each other.
+
+.. math::
```

Comment:
This math is missing something, no? 

Don't we need a Quantifier on top of Line 121. I would guess you do that operation **\forall** users? And then? Take an average? (but I see that there is an average() there too...). 

I feel this math needs to be a bit more precise. 

(Btw, if you end up changing it, plz make sure to update other places where we have the same explanation. I never figured out a good way of not duplicating these explanations between the readme, code comments, and HTML side.)

Reply:
Got it, could further revise some notions and clarify.